### ISSUE TITLE
Floating Action Button problem in ver 0.70

### ISSUE THREAD

**Issue by jonbhanson** [stryder-dev/flutter_platform_widgets#213]
I use this code to show a FAB on the first page (page 0) of a bottom navigation, but not show any FAB's on the other pages.  This code worked in version 0.60.1, but in 0.70.x, the FAB button from page 0 is shown on the rest of the pages.  Do I need to make an update to this code to remove the FAB on pages 1-3?

```
    floatingActionButtons = {
      0: FloatingActionButton(child: Icon(Icons.add)),
      1: null,
      2: null,
      3: null,
    };

    return PlatformTabScaffold(
      //...
      tabController: tabController,
      material: (context, platform) =>
          MaterialTabScaffoldData(floatingActionButton: floatingActionButtons[tabController.index(context)]),
    )
```

**Comment by aqwert** [comment]
I see what you mean. The material property is passed in above the AnimatedWidget so any changes in tabs does not re-get the material data. It could be solved (I think) by having another property called `materialBuilder` which will also pass in the current index.

```
 return PlatformTabScaffold(
      //...
      tabController: tabController,
      materialBuilder: (context, platform, index) =>
          MaterialTabScaffoldData(floatingActionButton: floatingActionButtons[index]),
    )
```

**Comment by jonbhanson** [comment]
Yes, @aqwert , I think that would be a solution.

**Comment by aqwert** [comment]
Fixed in v0.72.0 sorry took so long

**Comment by jonbhanson** [comment]
Thanks so much @aqwert !
